src/overlays/actors/ovl_Obj_Syokudai/z_obj_syokudai.h

```diff
@@ -7,12 +7,32 @@ struct ObjSyokudai;
 
 typedef struct ObjSyokudai {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ ColliderCylinder colliderCylinder1;
-    /* 0x190 */ ColliderCylinder colliderCylinder2;
-    /* 0x1DC */ s16 unk_1DC;
-    /* 0x1DE */ char unk_1DE[0x16];
+    /* 0x144 */ ColliderCylinder standCollider;
+    /* 0x190 */ ColliderCylinder flameCollider;
+    /* 0x1DC */ s16 litTimer;
+    /* 0x1DE */ u8 flameTexScroll;
+    /* 0x1DF */ s8 pendingAction;
+    /* 0x1E0 */ LightNode* lightNode;
+    /* 0x1E4 */ LightInfo lightInfo;
 } ObjSyokudai; // size = 0x1F4
 
 extern const ActorInit Obj_Syokudai_InitVars;
 
+typedef enum {
+    /* 0 */ OBJ_SYOKUDAI_TYPE_SWITCH_CAUSES_FLAME,
+    /* 1 */ OBJ_SYOKUDAI_TYPE_FLAME_CAUSES_SWITCH,
+    /* 2 */ OBJ_SYOKUDAI_TYPE_NO_SWITCH
+} ObjSyokudaiTypes;
+
+typedef enum {
+    /* -1 */ OBJ_SYOKUDAI_PENDING_ACTION_CUTSCENE_NO_SWITCH = -1,
+    /*  0 */ OBJ_SYOKUDAI_PENDING_ACTION_NONE,
+    /*  1 */ OBJ_SYOKUDAI_PENDING_ACTION_CUTSCENE_AND_SWITCH
+} ObjSyokudaiPendingActions;
```

Comment:
I wonder if we could chop of a few words of this :thinking:  Maybe remove the "PENDING"?
```suggestion
typedef enum {
    /* -1 */ OBJ_SYOKUDAI_ACTION_CUTSCENE_NO_SWITCH = -1,
    /*  0 */ OBJ_SYOKUDAI_ACTION_NONE,
    /*  1 */ OBJ_SYOKUDAI_ACTION_CUTSCENE_AND_SWITCH
} ObjSyokudaiPendingActions;
```

Reply:
I feel like pending describes it better, personally. The pending action gets set at the bottom of the update func, and then the top of the func checks for a pending action, executes it, and then sets the pending action to NONE. But I do agree the names are a little long.

Reply:
Alternatively, i could get rid of "_NO_SWITCH" at the end of the first one and just have it be implicit. I thought about doing it, but I figured making it clear that this was the nonswitching case was more important than brevity.